### ISSUE TITLE
Publish helm chart and Docker image to ArtifactHub

### ISSUE THREAD

**Issue by christianhuth** [zulip/docker-zulip#359]
### Pitch
The helm chart should be published so we can install it with classic helm command without having to clone the whole repository. Also this will enable to list it at https://artifacthub.io/.

We could use the official tool https://github.com/helm/chart-releaser in the CI to publish it.

**Comment by varac** [comment]
This would be great also for visibility. When I don't find a chart on https://artifacthub.io I often don't look further because I don't want to bother with half-baked charts which don't step in the public ring so to say.
Any chance to tackle this soon ?

**Comment by christianknell** [comment]
I totally agree with you. Not being visible at the [ArtifactHub](https://artifacthub.io) is a big downer. I would love to tackle this issue, but i need maintainer rights to set-up GitHub Pages.

**Comment by alexmv** [comment]
Keeping this separate from #407, since it's primarily about ArtifactHub.
